pkgs/by-name/cr/crowdsec-firewall-bouncer/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "crowdsec-firewall-bouncer";
+  version = "0.0.33";
+
+  src = fetchFromGitHub {
+    owner = "crowdsecurity";
+    repo = "cs-firewall-bouncer";
+    tag = "v${version}";
+    hash = "sha256-4fxxAW2sXGNxjsc75fd499ciuN8tjGqlpRIaHYUXwQ0=";
+  };
+
+  vendorHash = "sha256-Bhp6Z2UlCJ32vdc3uINCGleZFP2WeUn/XK+Q29szUzQ=";
+
+  ldflags = [
+    "-X github.com/crowdsecurity/go-cs-lib/version.Version=${src.tag}"
+  ];
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+
+  versionCheckProgram = "${placeholder "out"}/bin/cs-firewall-bouncer";
+  versionCheckProgramArg = "-version";
+  doInstallCheck = true;
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = with lib; {
+    homepage = "https://crowdsec.net/";
+    changelog = "https://github.com/${src.owner}/${src.repo}/releases/tag/${src.tag}";
```

```suggestion
    changelog = "https://github.com/crowdsecurity/cs-firewall-bouncer/releases/tag/v${finalAttrs.version}";
```

don't refer to `src` here since this would break in case someone wants to override `src` with something that doesn't have these attributes (e.g. a local path)

---

Ahh whoops, that’s my bad.